tests/test_database.py

```diff
@@ -0,0 +1,158 @@
+import unittest
+from unittest.mock import patch, MagicMock
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+import sys
+import os
+
+# Add the parent directory to the Python path
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+from src.database import (
```

Comment:
imports should always be above all operations

Reply:
Noted